stable/traefik/templates/storeconfig-job.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if and .Values.kvprovider.type .Values.kvprovider.storeAcme }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "storeconfig-job"
+  annotations:
+    "helm.sh/hook": post-install
```

Comment:
Should this be a `pre-install` hook? We need this present before the Traefik pods come online, right? [The (admittedly sparse) documentation implies that it should be run before Traefik is started](https://docs.traefik.io/user-guide/cluster/#deploy-a-trfik-cluster). That being said, if you _were_ to deploy `etcd`, `consul`, etc alongside this I imagine a preinstall hook would get you into trouble. 

That

Reply:
Nope, if we make this pre-install we can't install in a master chart consul/etcd or sth. similar as the hook prevents further install process.

On the other hand traefik may fail once, but for me staring the server is slower than running the storeconfig job:

```
traefik-ingress-844f599647-bjcfg          1/1       Running     0          4d
traefik-ingress-844f599647-jxwxm          1/1       Running     0          4d
traefik-ingress-844f599647-kqhtm          1/1       Running     0          4d
```

Don't know if its just because of my cluster but this approach seems to be the most stable one.

The worst case which can happen is that the container gets rescheduled after the storeconfig job succeeded.

Reply:
makes sense to me 👍  -- thank you for the detailed response!